portfolio/.theia/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "java.configuration.updateBuildConfiguration": "interactive"
```

Comment:
You don't need to check in this file. You can add it to .gitignore so git just ignores it.